packages/alias/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.1",
+  "version": "0.0.0",
```

Comment:
I'm not fan of that because dev(maintainer) should know on which version they are working.

Reply:
I was thinking about this, but why? What would you use that info for, and how often would you need it? Also, you can easily get that from git by checking its latest tag.

Committing the version bump within the workflow adds a rather significant amount of complexity that I'm thinking is not worth the trouble.

Reply:
why not make it by humain ? like https://github.com/nodejs-loaders/nodejs-loaders/pull/24/files

Reply:
I think you're missing the fundamental problem: releasing from a commit that is not the latest means you effectively can't change this value (you can commit it without a bunch of tedious git gymnastics). I voided it here to zero so it is obviously not the current version (otherwise, it would effectively stall at the current and never change going forward).